toolkit/models.py

```diff
@@ -2,16 +2,16 @@
 
 
 # Create your models here.
-class Account(models.Model):
-    """Creates accounts
+class User(models.Model):

```

Comment:
we could keep this as Account to not confuse it from the User base class.

Reply:
What do we get out of that considering the base User Class has Username, Email and password as options

Reply:
The only issue it brings is if we ever have to reference the original User class, then it will get mad at us for having two User classes